src/Components/EmployeeList.js

```diff
@@ -1,13 +1,38 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
 import Employee from "./Employee";
 import "./EmployeeList.css";
 
 export const EmployeeList = () => {
+  const [employees, setEmployees] = useState([]);
+  function makeFetchRequest() {
+    fetch("https://vet-resource-api-9-2.herokuapp.com/api/employees")
+      .then((response) => response.json())
+      .then((response) => setEmployees(response));
+  }
+
+  useEffect(() => {
+    makeFetchRequest();
+  }, []);
+
   return (
     <main>
       <h2>All Staff</h2>
       <section className="employee-list">
-        <Employee />
+        {employees.map(
+          ({ firstName, lastName, prefix, postfix, title, id }) => {
+            return (
+              <Employee
```

Comment:
Could just pass the entire employee object to the Employee component, as `<Employee employee={employee} />` or something, that way you don't have to break down every key

Reply:
Understood